tests/FSharp.Core.UnitTests/FSharp.Core/Microsoft.FSharp.Core/IntConversionsTestGenerator.fsx

```diff
@@ -45,7 +45,7 @@ do
 // This file is automatically generated by IntConversionsTestGenerator.fsx    
 namespace FSharp.Core.UnitTests        
 open System                           
-open NUnit.Framework
+open XUnit
 open FSharp.Core.UnitTests.LibraryTestFx
```

Comment:
This here is not test code, but test generation code. This file is `IntConversionsTestGenerator.fsx` and the actual tests are located in [IntConversionsGenerated](https://github.com/dotnet/fsharp/blob/85617ced7e1d5aac0dd132c1fa973d5587c52927/tests/FSharp.Core.UnitTests/FSharp.Core/Microsoft.FSharp.Core/IntConversionsGenerated.fs).

Things were initially Nunit here and were generating Nunit. But 4 years ago this [migrated to](https://github.com/dotnet/fsharp/pull/9992/files#diff-ffa136487f3031a2cc3d55cf756257c200afa7705ac7cbf6f36d70c1a0134e8e) to Xunit. The generated file was updated, the generator was not - hence doing it here.